src/solver/utils/ortools_utils.cpp

```diff
@@ -1,538 +1,544 @@
+      = "cplex_mip"; // TODO JMK : not defined in current ortools RTE branch.
+
+    _solverLinearProblemOptimStringMap[OrtoolsSolver::gurobi] = "gurobi_lp";
+    _solverMixedIntegerProblemOptimStringMap[OrtoolsSolver::gurobi] = "gurobi_mip";
+
+    _solverLinearProblemOptimStringMap[OrtoolsSolver::glpk] = "glpk_lp";
+    _solverMixedIntegerProblemOptimStringMap[OrtoolsSolver::glpk] = "glpk_mip";
+
+    _solverLinearProblemOptimStringMap[OrtoolsSolver::glop_cbc] = "glop";
+    _solverMixedIntegerProblemOptimStringMap[OrtoolsSolver::glop_cbc] = "cbc";
+
+    /* TODO JMK : see how we can get optimization problem type with current ortools RTE branch
+    (can't use enum because of compile switch)
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::sirius]       =
+    MPSolver::OptimizationProblemType::SIRIUS_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::sirius] =
+    MPSolver::OptimizationProblemType::SIRIUS_MIXED_INTEGER_PROGRAMMING;
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::coin]       =
+    MPSolver::OptimizationProblemType::CLP_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::coin] =
+    MPSolver::OptimizationProblemType::CBC_MIXED_INTEGER_PROGRAMMING;
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::xpress]       =
+    MPSolver::OptimizationProblemType::XPRESS_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::xpress] =
+    MPSolver::OptimizationProblemType::XPRESS_MIXED_INTEGER_PROGRAMMING;
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::glop_scip]       =
+    MPSolver::OptimizationProblemType::GLOP_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::glop_scip] =
+    MPSolver::OptimizationProblemType::SCIP_MIXED_INTEGER_PROGRAMMING;
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::cplex]       =
+    MPSolver::OptimizationProblemType::CPLEX_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::cplex] =
+    MPSolver::OptimizationProblemType::CPLEX_MIXED_INTEGER_PROGRAMMING;
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::gurobi]       =
+    MPSolver::OptimizationProblemType::GUROBI_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::gurobi] =
+    MPSolver::OptimizationProblemType::GUROBI_MIXED_INTEGER_PROGRAMMING;
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::glpk]       =
+    MPSolver::OptimizationProblemType::GLPK_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::glpk] =
+    MPSolver::OptimizationProblemType::GLPK_MIXED_INTEGER_PROGRAMMING;
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::glop_cbc]       =
+    MPSolver::OptimizationProblemType::GLOP_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::glop_cbc] =
+    MPSolver::OptimizationProblemType::CBC_MIXED_INTEGER_PROGRAMMING;
+
+            */
+}
+
+bool OrtoolsUtils::isOrtoolsSolverAvailable(OrtoolsSolver ortoolsSolver)

```

Comment:
This should be the only place where there is changes.

Reply:
I passed dos2unix on that file, this solves the problem